src/main.rs

```diff
@@ -251,13 +251,19 @@ fn start_output_loop_thread(out_rx: Receiver<ChildEvent>) -> JoinHandle<()> {
                                     "{id}: Process exited with {} - will retry in {}",
                                     status, delay
                                 );
-                            } else {
+                            } else if delay == RetryDelay::NONE {
```

Comment:
It seems the newly added if statement here is either unnecessary or incorrect. Unnecessary because positive delay times are already captured above, incorrect because it leaves out the hypothetical negative delay times ("less that NONE"). What was the reason for adding it?